contrib/verify-commits/trusted-git-root

```diff
@@ -1 +1 @@
-82bcf405f6db1d55b684a1f63a4aabad376cdad7
+577bd51a4b8de066466a445192c1c653872657e2
```

Comment:
> Merge https://github.com/bitcoin/bitcoin/pull/25197: contrib: Remove keys that are no longer used for…

we need to dashify it.

```suggestion
3b7deea3b51a5c6b466a78bfbfcdfed48772a2f5
```

Reply:
Nah; these changes should be in new PRs; we don't actually use these scripts sooo... 🤷 feel free to open other PRs